neo/Network/P2P/Peer.cs

```diff
@@ -18,13 +18,15 @@ namespace Neo.Network.P2P
 {
     public abstract class Peer : UntypedActor
     {
-        public class Start { public int Port; public int WsPort; }
+        public class Start { public int Port; public int WsPort; public int MinDesiredConnections; public int MaxConnections; }
         public class Peers { public IEnumerable<IPEndPoint> EndPoints; }
-        public class Connect { public IPEndPoint EndPoint; }
+        public class Connect { public IPEndPoint EndPoint; public bool IsTrusted = false; }
         private class Timer { }
         private class WsConnected { public WebSocket Socket; public IPEndPoint Remote; public IPEndPoint Local; }
 
         private const int MaxConnectionsPerAddress = 3;
+        public const int DefaultMinDesiredConnections = 10;
+        public const int DefaultMaxConnections = DefaultMinDesiredConnections * 2;
```

Comment:
Let me know if we want to increase this default value to something higher. Might want to use * 4 (40)

Reply:
Interesting design, well defined, Jeff.

Reply:
Maybe 4 is interesting.

Reply:
I switched to * 4.